cns/restserver/internalapi_test.go

```diff
@@ -381,6 +381,32 @@ func TestReconcileNCWithSystemPods(t *testing.T) {
 	validateNCStateAfterReconcile(t, req, expectedNcCount, expectedAssignedPods)
 }
 
+func TestGetHomeAz(t *testing.T) {
+	expHomeAz := "01"
+	mnma := &fakes.NMAgentClientFake{
+		SupportedAPIsF: func(_ context.Context) ([]string, error) {
+			return []string{
+				"GetHomeAz",
+			}, nil
+		},
+		GetHomeAzF: func(_ context.Context) (nma.HomeAzResponse, error) {
+			return nma.HomeAzResponse{
+				HomeAz: expHomeAz,
+			}, nil
+		},
+	}
+	cleanup := setMockNMAgent(svc, mnma)
```

Comment:
why is the mockNMA named as cleanup?

Reply:
because the setMockNMAgent  return a cleanup function that will restore NMAgent back to what it was